### DISCUSSION TITLE
Amazon Prime Video problems

### DISCUSSION THREAD

**Discussion by Tobix99** [0xERR0R/blocky#153] [Q&A]
Hey,
I have some trouble with blocky and Amazon Prime Video. I'm on 2 **Unblocked** Devices (LG SmartTV and PC) and have the problem, that when I try to watch something on amazon Prime Video, the hole website loads and I can select a video, but the Player gives me an error, like: "An error occurred. try again later". When I switch my DNS over to like 1.1.1.1, then everything works. So my I use blocky atm without the blocking, so just caching. And in my logs all Querys where resolved and not blocked (or even cached). So I'm not sure where to search for the error.

Edit: Amazon Error message

> Video Unavailable
> We're experiencing a problem playing this video. For assistance, please go to www.amazon.de/videohelp.

**Comment by 0xERR0R** [comment]
Hey,
difficult to say, but may be it is a caching problem? Do you have some special settings regarding the min and max cache time? Maybe Amazon changes the IP to often.

**Comment by Tobix99** [comment]
Hey,
ok, so found the problem on my PC (Prime doesn't like Safari, with Chrome and Blocky it works now).

But on the LG Smart TV I had to change the DNS Server to get it working. Will look into it tomorrow and see if I catch something in the logs for the LG TV.

My Cache Settings are:
```
caching:
  minTime: 0
  maxTime: 0
```

It doesn't even start to play the Video (the Video Player on the Smart TV throws the error, not even the gallery where you can select the movie)

**Comment by 0xERR0R** [comment]
@Tobix99 any news?
